### DISCUSSION TITLE
React toast notification example?

### DISCUSSION THREAD

**Discussion by gouku** [floating-ui/floating-ui#2191] [React DOM]
I'd like to know if this library can be used as a toast notification component? If someone could provide an example would be great!

**Comment by atomiks** [comment]
Hi, not right now, however one of the goals of the "floating" UI adjective name is that it would potentially support things like Toasts or draggable elements in the future, and does already support dialogs that don't use anchor positioning. I'm going to open an issue for this feature. Thanks!
